packages/meta-updater/src/updatePackageJson.ts

```diff
@@ -111,13 +125,16 @@ async function getScripts(
     return scripts;
   }
 
-  const cleanDirs = ["./out", "tsconfig.tsbuildinfo"];
+  const cleanDirs = ["./out", "tsconfig.tsbuildinfo", "./dist", "./build"];
```

Comment:
Didn't seem worth the trouble of keeping different clean dirs for every package; easier to just remove a standard list. Can revisit if we have a real need for it in the future.